src/OpenLoco/src/Game.cpp

```diff
@@ -130,11 +133,6 @@ namespace OpenLoco::Game
     // 0x0043BFF8
     void loadGame()
     {
-        if (isTitleMode())
-        {
-            Title::stop();
```

Comment:
without this - when does this otherwise get called? it needs to be called somewhere before the save-game is actually loaded

Reply:
`loadGame()` is called again when a game is loaded, and it goes into:
```
else if (!isNetworked())
{
    if (Game::loadSaveGameOpen())
```
where `importSaveToGameState()` in the end calls `stopMusic()`.